src/interface/src/app/map/map-manager.ts

```diff
@@ -39,11 +40,13 @@ export class MapManager {
   constructor(
     private matSnackBar: MatSnackBar,
     maps: Map[],
+    mapViewOptions$: BehaviorSubject<MapViewOptions>,
     popupService: PopupService,
     startLoadingLayerCallback: (layerName: string) => void,
     doneLoadingLayerCallback: (layerName: string) => void
   ) {
     this.maps = maps;
```

Comment:
Nit: do you need to make these assignments in the constructor?  https://dev.to/satansdeer/typescript-constructor-shorthand-3ibd